docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+Any DID message uploaded to the Tangle, with the exception of the very first DID message that creates the DID, MUST contain a `previous_message_id` field. This field MUST carry the MessageId, an IOTA indexation for a single message, of the previous DID Document that is updated with this DID message. This value SHOULD be used to order DID messages during the resolving procedure. If two or more DID messages reference the same `previous_message_id` an ordering conflict is identified and is resolved using a deterministic ordering mechanism. 
+
+Example of an IOTA MessageId:
+```json
+"previous_message_id": "cd8bb7baca6bbfa1de7813bd1753a2de026b6ec75dba8a3cf32c0d4cf6038917"
+```
+
+### Signing Key
+
+DID Documents published to the Tangle must be cryptographically signed. As such the DID Document MUST include one verification method with a public key. It is recommended, for security reasons, to not use this keypair for other purposes as the control over this private key is vital for controlling the identity. It is RECOMMENDED to name this public key #_sign-x, where x is the index of the signing key, which is incredemented everytime the signing key is updated, starting at index 1. 
+
+### Autonomy of DID Integration Messages
+
+A DID Integration message MUST contain a valid DID Document according to the W3C DID standard. In addition, the message has further restrictions:
+
+* The DID Document MUST always contain one or more verification methods with a public key in the `verificationMethod` object of the DID Document. It is RECOMMENDED to only use this key for updating the DID Document and name this public key #_sign-x. 
+* The first DID Document in the chain MUST contain a `verificationMethod` that contains a public key that, when hashed using the `Blake2b-256` hashing function, equals the tag section of the DID. This prevents the creation of conflicting entry messages of the chain by adversaries.
+* An Integration DID message must be published to an IOTA Tangle on an index that is generated by the `BLAKE2b-256` of the public key, created in the [generation](#generation) event, encoded in `hex`. 
+* DID Integration messages SHOULD contain all cumulative changes from the Diff Chain associated to the last Integration Chain message. Any changes added in the Diff Chain that are not added to the new DID Integration message will be lost. 
+* DID Integration Messages have at least the following attributes:
+    * `previous_message_id` (REQUIRED): This field provides an immutable link to the previous DID document that is updated and is used for basic ordering of the DID messages, creating a chain. The value of `previous_message_id` MUST be a string that contains an IOTA MessageId from the previous DID message it updates, which MUST reference an Int chain message. The field SHOULD be ommited if the DID message is the start of the Int chain, otherwise the field is REQUIRED. Read the [Previous Message Id](#previous-message-id) section for more information. 
```

Comment:
OOC, I noticed that the other properties of DID Documents use camelCase, was it intentional that the IOTA-specific properties use lower_snake_case?

Reply:
That's actually a mistake. We should update everything to camelCase due to that being the standard for JSON. Feel free to open up an issue (Otherwise I will)

Reply:
Created #222

Reply:
Updated it in the spec